README.md

```diff
@@ -1 +1,124 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [My Catalog App] <a name="about-project"></a>
+
+### Video description
+
+[CLick to watch video](https://drive.google.com/file/d/1YLngZXukysB0ou5tGQvS_QXsk7qR7OG9/view?usp=sharing)
+
+**[My Catalog App]** is a project that shows you how Object Oriented Programming can be used in Ruby
+
+## 🛠 Built With <a name="built-with">Ruby</a>
+
+### Tech Stack <a name="tech-stack">Ruby</a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.ruby-doc.org/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- [] List all books'
+- [] List all music albums'
+- [] List of games'
+- [] List all labels'
+- [] List all genres'
+- [] List all authors'
+- [] List all movies'
+- [] Add a book'
+- [] Add a music album'
+- [] Add movie'
+- []Add a game'
+- []Add an author'
+- []Exit'
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- [ ] Clone the git repository
+
+### Prerequisites
+
+In order to run this project you need:
+
+- to have ruby installed on your system
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/martinkarugaba/oop-school-library
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Martin Karugaba**
+
+- GitHub: [martinkarugaba](https://github.com/martinkarugaba)
+- Twitter: [@martin_karugaba](https://twitter.com/martin_karugaba)
+- LinkedIn: [martinkarugaba](https://www.linkedin.com/in/martinkarugaba/)
+
+👤 **Ahmed**
+
+- GitHub: [newtonah0](https://github.com/newtonah0)
+
+👤 **Enoh Collins**
+
+- GitHub: [enocol](https://github.com/enocol)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [] Update Catalog with new details. - eg Update book
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, consider giving it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the project inspiration.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly please ensure that your license file is properly referenced, the link it broken because there is a typo error with the name. 

<img width="534" alt="Untitled" src="https://github.com/martinkarugaba/my-catalog/assets/103386183/65494f36-f28d-4ee6-b7e5-3dd925864350">